RPIPs/RPIP-44.md

```diff
@@ -18,16 +18,36 @@ This proposal specifies when [execution layer triggerable exits as defined in EI
 There is at least one additional use case not addressed in this RPIP, which is to exit badly-performing (eg, abandoned) validators. It is likely the pDAO will wish to supersede this RPIP to include that functionality. However, as it requires modeling and is not a critical component of the tokenomics rework, it is not being addressed at this time. For similar reasons, this RPIP does not include a keeper-based design to reward those that trigger forced exits. Please see a [historical version of RPIP-44](https://github.com/rocket-pool/RPIPs/blob/09d445accaa77f355acae1e943910ad0229a1d2e/RPIPs/RPIP-44.md) for initial but incomplete ideas to address abandonment and a keeper network.
 
 ## Specification
-- An `exit` function SHALL be included in the Megapool delegate
-  - This function SHALL take in an array of validator public keys
-  - This function MAY be freely called by the Node Operator
-  - This function MAY be called by any user if the Node has a `deficit` greater than `exit_deficit`
-    - It MUST NOT be possible to exit more than the minimum number of validators needed to cover `deficit`
-- `debt` is incurred via penalties, as enumerated in [RPIP-42](RPIP-42.md)
-- The protocol SHALL use `rewards` and/or `credit` held by the NO to decrease `debt` by the same amount prior to taking action on exits
-- The protocol SHOULD use staked RPL held by the NO to decrease `debt` by the corresponding amount prior to taking action on exits
-- The initial settings SHALL be:
-  - `exit_deficit`: 0.2 ETH
+
+This specification extends the specification of megapools in
+[RPIP-43](RPIP-43.md) with a facility for removal of validators from the
+megapool to be done by parties other than the Node Operator.
+
+Funds (ETH) associated with a megapool SHALL be accounted in (at least) the following categories:
+- Credits (`credit`): Funds that can be used towards validator deposits; sources include but are not limited to deposits from the Node Operator, ETH staked on behalf of the node ([RPIP-32](RPIP-32.md)), and ETH already staked on the beacon chain for a migrating validator
+- Deposits (`deposited`): Funds initially received as credits that are currently staked on the beacon chain for validators associated with this megapool
+- Withdrawals (`withdrawn`): Funds received into the megapool via withdrawals from the beacon chain, including both principal and the node's share of rewards, NOT including pool stakers' share of the rewards
+- Penalties (`debt`): Funds lost to penalties as enumerated in [RPIP-42](RPIP-42.md), or repaid from other balances as specified in this RPIP
+
+The following quantity, `deficit`, is derived based on the megapool's balances:
```

Comment:
Hmmm... megapool is a smart contract and the balance of that is defined. I'd probably go with different wording "derived based on the categorized funds above" or something.